Documentation/tectonic.md

```diff
@@ -0,0 +1,52 @@
+# Rook on Tectonic
+
+Here is a running guide on how to implement Rook on Tectonic. A complete guide on how to install Tectonic is out of the scope of the Rook project. But many info can be found on the [Tectonic website](https://coreos.com/tectonic/docs/latest/)
+
+## Prequisites
+
+- An installed tectonic-installer. These steps are described on [the Tectonic website](https://coreos.com/tectonic/docs/latest/install/bare-metal/#4-tectonic-installer)
+- A running matchbox node which will do the provisioning (Bare metal)
+- You can run through all steps of the GUI installer, but in the last step, choose `Boot manually`. This way we can make the necessary changes first.
+
+## Edit the kubelet.service file to be compatible with Rook
+
+To make the kubelet compatible with Rook we need to make a few adaptions to the systemd service file generated by the Tectonic-installer.
+
+First cd into the directory where you untarred the tectonic installer and find your newly generated cluster configuration files.
+
+```
+ $ cd ~/tectonic/tectonic-installer/LINUX-OR-DARWIN/clusters
+ $ nano modules/ignition/resources/services/kubelet.service
+```
+
+And after the line `ExecStartPre=/bin/mkdir -p /var/lib/cni`, paste the following extra lines:
```

Comment:
Writing to put it after the last `mkdir` and/or `ExecStartPre=` line should be enough, when that line changes this will be "outdated".